specreduce/tests/test_tracing.py

```diff
@@ -144,15 +144,15 @@ def test_fit_trace():
     try:
         FitTrace(img_win_nans, guess=guess, window=window)
     except ValueError as e:
-        print(f"All-NaN window error message: {e}")
+        print(f"(expected) All-NaN window error message: {e}")
     else:
         raise RuntimeError('Trace was erroneously calculated on all-NaN window')
 
     # error on trace of all-nan image
     try:
         FitTrace(img_all_nans)
     except ValueError as e:
-        print(f"All-NaN image error message: {e}")
+        print(f"(expected) All-NaN image error message: {e}")
     else:
         raise RuntimeError('Trace was erroneously calculated on all-NaN image')
```

Comment:
would this test be simplified by using `pytest.raises`?

Reply:
Yes, it would. I just committed an update.